app/oauth.go

```diff
@@ -896,8 +896,12 @@ func (a *App) AuthorizeOAuthUser(w http.ResponseWriter, r *http.Request, service
 		mlog.Error("Error getting OAuth user", mlog.Int("response", resp.StatusCode), mlog.String("body_string", bodyString))
 
 		if service == model.ServiceGitlab && resp.StatusCode == http.StatusForbidden && strings.Contains(bodyString, "Terms of Service") {
+			url, err := url.Parse(*sso.UserAPIEndpoint)
+			if err != nil {
+				return nil, "", stateProps, nil, model.NewAppError("AuthorizeOAuthUser", model.NoTranslation, nil, err.Error(), http.StatusInternalServerError).Wrap(errors.Wrapf(err, "error parsing %s", *sso.UserAPIEndpoint))
```

Comment:
Please remove "err.Error()", and use an empty string. We are wrapping the whole error again with `.Wrap`

Reply:
@agnivade fixed this